spec/requests/users/sign_up_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'GET /users/sign_up', type: :request do
+  subject { response }
+
+  context 'when user signs up' do
+    before do
+      post user_registration_path, params: {
```

Comment:
instead of `'user[...]'` as keys for the params hash, you can do

```
params: {
  user: {
    ...
  }
}
```

and Rails will convert to what you want ;)